notifications_utils/jinja_templates/email_preview_template.jinja2

```diff
@@ -34,33 +34,33 @@
     </tbody>
   </table>
 
-  <div class="email-message-body w-full m-0 relative break-words clear-both box-border px-doubleGutter py-4 border-gray-grey2 border">
+  <div class="email-message-body w-full relative break-words box-border px-doubleGutter border-gray-grey2 border">
 
-    <div style="max-width: 580px; margin: 0 auto">
+    <div style="max-width: 580px; margin: 0 auto;">
       {% if fip_banner_english %}
-        <div style="max-width: 326px">
+        <div style="margin: 20px auto 30px auto;">
```

Comment:
Margins are defined in the email preview but we use a div with a specified height in the real email template because some email clients don't support margins